avro-builder/builder/src/main/java/com/linkedin/avroutil1/builder/operations/codegen/own/AvroUtilCodeGenOp.java

```diff
@@ -91,10 +93,53 @@ public void run(OperationContext opContext) throws Exception {
     long parseEnd = System.currentTimeMillis();
 
     Map<String, AvscParseResult> allNamedSchemas = context.getAllNamedSchemas();
+
+    // Handle duplicate schemas
     Map<String, List<AvscParseResult>> duplicates = context.getDuplicates();
+    for (Map.Entry<String, List<AvscParseResult>> duplicateEntry : duplicates.entrySet()) {
+      String fqcn = duplicateEntry.getKey();
+      StringBuilder allFilesString = new StringBuilder();
+      for (AvscParseResult avscParseResult : duplicateEntry.getValue()) {
+        allFilesString.append(avscParseResult.getURI());
+        allFilesString.append(", ");
```

Comment:
nit - use StringJoiner

Reply:
more nit - if schema isnt top level in the file would be nice to also print the line where its defined

Reply:
@radai-rosenblatt 
> more nit - if schema isnt top level in the file would be nice to also print the line where its defined
In `avro-schemas`, each FQCN that they check for duplicates against corresponds to a file. I assumed that each `AvscParseResult` in `context.getDuplicates()` also corresponds to a file. Is that correct? If so, top level should be fine

Reply:
If that is not the case in AVRO_UTIL, do we only want to compare top-level duplicates or should we also compare nested duplicates?

Reply:
we want to find duplicates everywhere - top level or not, importable or not